merlin/study/study.py

```diff
@@ -107,6 +107,13 @@ def __init__(
             "MERLIN_SOFT_FAIL": str(int(ReturnCode.SOFT_FAIL)),
             "MERLIN_HARD_FAIL": str(int(ReturnCode.HARD_FAIL)),
             "MERLIN_RETRY": str(int(ReturnCode.RETRY)),
+            # below will be substituted for sample values on execution
+            "MERLIN_SAMPLE_VECTOR": " ".join([ "$({})".format(k) for k in self.get_sample_labels(from_spec=self.original_spec)]),
+            "MERLIN_SAMPLE_NAMES": " ".join(self.get_sample_labels(from_spec=self.original_spec)),
+            "MERLIN_SPEC": os.path.join(
```

Comment:
These are good additions, I think we should add options for all three spec files in the

Reply:
The expanded yaml is the one that is actually RUN, maybe RUN should be changed to PARTIAL.

Reply:
How about
MERLIN_SPEC_ORIGINAL
MERLIN_SPEC_REQUESTED
MERLIN_SPEC_RUN
??? Other votes? not sure here

Reply:
I vote for
MERLIN_SPEC_ORIGINAL
MERLIN_SPEC_PARTIAL
MERLIN_SPEC_RUN

so that the names are mostly similar.

Reply:
Yeah I don't really like "partial" b/c it's not very descriptive. It's really the original with any command-line override variables substituted. I can't think of a good word tho.

Reply:
NOSAMPLES ?

Reply:
EXPANDED_VARS
EXPANDED_ALL?

Reply:
Ok, I've (ok @ryannova ) have decided on
MERLIN_SPEC_ORIGINAL_TEMPLATE
MERLIN_SPEC_EXECUTED_RUN
MERLIN_SPEC_ARCHIVED_COPY

Reply:
LGTM

Reply:
agreed. I put @ryannova on it